### PR TITLE
assign-reviewer.py script with comments

### PR DESCRIPTION
**Pull request by pmandler**
Looks like there are additions in ISWC that I may want to grab.

### CODE REVIEW

File: samples/python/assign-reviewers.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+parser = argparse.ArgumentParser()
+parser.add_argument('assignments', help="either (1) a csv file containing reviewer assignments or (2) a string of the format '<email_address>,<paper#>' e.g. 'reviewer@cs.umass.edu,23'")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
```
Comment (mspector) [inline samples/python/assign-reviewers.py]: you should be able to remove the "if/else" block here, and just have a single line like this one

**Comment by mspector** [comment]
Which additions in IWSC are you referring to? Some of the code in the scripts are specific to DeSemWeb, and won't necessarily apply to other conferences...

File: samples/python/assign-reviewers.py (hunk 1 above)
Comment (pmandler) [inline samples/python/assign-reviewers.py]: I just saw the extra arguments that allowed for to add or remove.  And it looks like it handles user id's or emails.  This one is specific to email addresses.

Comment (mspector) [inline samples/python/assign-reviewers.py]: Ok, I see. Do you want to add those parameters before we approve this PR?

Comment (pmandler) [inline samples/python/assign-reviewers.py]: I think later.  Unlike that script which puts all the reviewers into one AnonReviewer group, this one would have to find the AnonReviewer# to remove it from the Reviewers group.

**Pull request merged**